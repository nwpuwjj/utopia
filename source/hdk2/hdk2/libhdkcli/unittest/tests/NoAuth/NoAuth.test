#
# NoAuth.test
#   Test that HTTP authentication is not included for methods which do not require it.
#

method: http://cisco.com/Unittest/NoAuthMethod
host: http://127.0.0.1:8080
