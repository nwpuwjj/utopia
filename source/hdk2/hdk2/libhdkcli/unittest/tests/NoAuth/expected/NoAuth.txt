+++++++ Request Headers +++++++
HTTP/1.1 POST /HNAP1
Connection: Close
Content-Type: text/xml; charset="utf-8"
Content-Length: 268
SOAPAction: "http://cisco.com/Unittest/NoAuthMethod"
Host: 127.0.0.1:8080

+++++++ Request Data +++++++
<?xml version="1.0" encoding="utf-8"?>
<Envelope xmlns="http://schemas.xmlsoap.org/soap/envelope/">
<Body>
<NoAuthMethod xmlns="http://cisco.com/Unittest/">
<StringArg0>The string argument for the NoAuthMethod Unittest</StringArg0>
</NoAuthMethod>
</Body>
</Envelope>


------- Response -------
HTTP/1.1 200 OK
Content-Type: text/xml; charset="utf-8"
Connection: close

<?xml version="1.0" encoding="utf-8" ?>
<soap:Envelope xmlns:soap="http://schemas.xmlsoap.org/soap/envelope/">
<soap:Body>
  <NoAuthMethodResponse xmlns="http://cisco.com/Unittest/">
      <NoAuthMethodResult>OK</NoAuthMethodResult>
  </NoAuthMethodResponse>
</soap:Body>
</soap:Envelope>

******* Client Result ******

      SOAP action: http://cisco.com/Unittest/NoAuthMethod
        HTTP host: http://127.0.0.1:8080
Network Object ID: (null)
    HTTP username: (null)
    HTTP password: (null)
          timeout: 0
       HNAP input: NoAuth.input.xml

<?xml version="1.0" encoding="utf-8"?>
<Envelope xmlns="http://schemas.xmlsoap.org/soap/envelope/">
<Body>
<NoAuthMethodResponse xmlns="http://cisco.com/Unittest/">
<NoAuthMethodResult>OK</NoAuthMethodResult>
</NoAuthMethodResponse>
</Body>
</Envelope>
malloc_interposer.c - ************************************************************
malloc_interposer.c - exit: malloc called 71 times
malloc_interposer.c - exit: free called 71 times
malloc_interposer.c - exit: malloc current = 0
malloc_interposer.c - exit: malloc total = 18180
malloc_interposer.c - exit: malloc max = 11078
malloc_interposer.c - ************************************************************
